Source/dump.f90

```diff
@@ -9242,7 +9242,14 @@ REAL(EB) FUNCTION SOLID_PHASE_OUTPUT(NM,INDX,Y_INDEX,Z_INDEX,PART_INDEX,OPT_WALL
          ENDDO SH_MATL_LOOP
          SOLID_PHASE_OUTPUT = SOLID_PHASE_OUTPUT * 0.001_EB
       ENDIF
-
+   CASE(76) ! CONVECTIVE HEAT FLUX GAUGE
+         IF (PY%HEAT_TRANSFER_COEFFICIENT>=0._EB) THEN
```

Comment:
More tab issues?  Please set you editor to use spaces only.

Reply:
My editor is set up for spaces only, but when I committed the updates there were some pre-existing tabs which my editor replaced with spaces. I reverted those back to the tabs which were already there thinking they were intentional. Do you want me to commit with those converted to spaces?